src/Components/CommentsList.jsx

```diff
@@ -0,0 +1,35 @@
+import { useEffect, useState } from "react";
+import { getCommentsByArticleId } from "../Utils/api";
+import { CommentCard } from "./CommentCard";
+
+export const CommentsList = ({ article_id }) => {
+  const [comments, setComments] = useState([]);
+  const [loading, setLoading] = useState(true);
+
+  useEffect(() => {
+    setLoading(true);
+
+    getCommentsByArticleId(article_id)
+      .then((comments) => {
+        setComments(comments);
+        setLoading(false);
+      })
+      .catch((error) => {
+        console.error("Error fetching comments:", error);
```

Comment:
Instead of logging here, please set up a new state for errors and display this message to the user in the page in case of an error.